packages/effects-core/src/plugin-system.ts

```
@@ -92,29 +91,27 @@ export class PluginSystem {
 
-  createPluginItem (name: string, props: VFXItemProps, composition: Composition): VFXItem {
-    const CTRL = pluginCtrlMap[name];
-
-    if (!CTRL) {
-      throw new Error(`The plugin '${name}' does not have a registered constructor.`);
-    }
-    const engine = composition.getEngine();
-    const item = new CTRL(engine, props, composition);
-
-    item.composition = composition;
+  async processRawJSON (json: spec.JSONScene, options: SceneLoadOptions): Promise<void[]> {
```

Biome

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)